modules/virt-restarting-vm-web.adoc

```diff
@@ -23,14 +23,12 @@ To avoid errors, do not restart a virtual machine while it has a status of *Impo
 
 * To stay on this page, where you can perform actions on multiple virtual machines:
 
-.. Click the Options menu {kebab} located at the far right end of the row.
+.. Click the Options menu {kebab} located at the far right end of the row and click *Resume VirtualMachine*.
 
 * To view comprehensive information about the selected virtual machine before
 you restart it:
 
 .. Access the *VirtualMachine details* page by clicking the name of the virtual
 machine.
 
-.. Click *Actions* -> *Restart*.
-
-. In the confirmation window, click *Restart* to restart the virtual machine.
+.. Click *Actions* -> *Start*.
```

Comment:
```suggestion
.. Click *Actions* -> *Restart*.
```

Reply:
@ousleyp For this one, I am seeing the **Restart** option grated-out. The user can only click **Start**. Is this a bug?? 

![image](https://github.com/openshift/openshift-docs/assets/25209155/9046dd6c-e311-4af0-9213-78dfdc193ec2)

Reply:
my other comment applies here too

Reply:
@ousleyp Thanks for verifying!